ner/src/main/java/edu/illinois/cs/cogcomp/ner/LbjTagger/NEWord.java

```diff
@@ -107,6 +107,19 @@ public NEWord(Word w, NEWord p, String type) {
         neTypeLevel1 = null;
     }
 
+    /**
+     * Creates dummy NEWord with empty string
+     */
+    public NEWord() {
+        super("");
+        label = "";
+        form = "";
+        originalForm = "";
+        neLabel = "";
+        neTypeLevel1 = "";
+        parts = new String[0];
+    }
+
```

Comment:
Good catch.  This was the result of a painful 3-way merge with my fork. Removing it...